app/components/Navigation/SeriesNavigation.js

```diff
@@ -1,27 +1,75 @@
-import React, {PropTypes} from 'react';
+import React, {PropTypes, Component} from 'react';
+import {View} from 'react-native';
 import {connect} from 'react-redux';
-import * as actionCreators from '../../actions/modal';
+import {showModal} from '../../actions/modal';
+import {showFilter, hideFilter, sortSeries} from '../../actions/series';
 import SeriesList from '../Series/SeriesList';
 import NavigatorWrapper from './NavigatorWrapper';
 import AddShowWizardWrapper from '../AddShowWizard/AddShowWizardWrapper';
+import {bindActionCreators} from 'redux';
+import BottomPicker from '../Widgets/BottomPicker';
+import {SERIES_FILTER} from '../../constants/variables';
 
+class SeriesNavigation extends Component {
 
-const SeriesNavigation = ({showModal}) => {
-  const initialRoute = {
-    name: 'Series',
-    index: 0,
-    component: SeriesList,
-    onPress: () => { showModal(<AddShowWizardWrapper />); },
-    rightText: 'Add',
-  };
-  return (
-    <NavigatorWrapper initialRoute={initialRoute} />
-  );
-};
+  constructor() {
+    super();
+    this.state = {
+      selectedFilter: SERIES_FILTER.get(0),
+    };
+  }
+
+  render() {
+    const initialRoute = {
+      name: 'Series',
+      index: 0,
+      component: SeriesList,
+      onPress: () => { this.props.showModal(<AddShowWizardWrapper />); },
+      onLeftPress: () => { this.props.showFilter(); },
```

Comment:
this can also be:

```js
 onLeftPress: () => this.props.showFilter(),
```

which is equivalent to 
```js
function() {
   return this.props.showFilter()
}
```